src/generate-common-seo.js

```diff
@@ -48,10 +48,13 @@ export function generateStructuredData(config = {}) {
   const { "theme-attributes":themeConfig, "social-links":socialLinks, "seo-metadata":seoMetadata = [] } = config;
   const homePageSeo = seoMetadata.find(page => page["owner-type"] === "home") || {};
   const { "page-title":pageTitle = "", description = "", keywords  =  "" } = get(homePageSeo, ["data"], {});
-
+  let enableNewsArticle = themeConfig['structured_data_news_article'];
```

Comment:
Variable name as it is does not convey adequate information, can be more verbose.

```suggestion
  let enableStructuredDataForNewsArticle = themeConfig['structured_data_news_article'];
```